File: Java/프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.java
class Solution {
    public int solution(String my_string) {
        char ch[] = my_string.toCharArray();
        int answer = 0;
        for(int i=0; i<ch.length ;i++){
            if(ch[i]>=48 && ch[i]<=57)
                answer+=Character.getNumericValue(ch[i]);
        }
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.js
function solution(my_string) {
    var answer = 0;
    for(var i=0; i<my_string.length ; i++){
        if(my_string.charAt(i)>='0'&&my_string.charAt(i)<='9'){
            answer+=Math.floor(my_string.charAt(i));
        }
    }
    return answer;
}